Directory.Packages.props

```diff
@@ -23,7 +23,7 @@
     <PackageVersion Include="NUnit.Analyzers" Version="*" />
     <PackageVersion Include="NUnit3TestAdapter" Version="*" />
     <PackageVersion Include="PolySharp" Version="1.15.0" />
-    <PackageVersion Include="Qowaiv" Version="7.2.3" />
+    <PackageVersion Include="Qowaiv" Version="7.2.1" />
```

Comment:
Why the downgrade?

Reply:
To prevent to have to deploy new versions for the other packages as well. It was a unthoughtful change in the first place.